packages/dev/core/src/Physics/v2/physicsEngineV2.ts

```diff
@@ -0,0 +1,153 @@
+import type { Nullable } from "../../types";
+import { Vector3 } from "../../Maths/math.vector";
+import type { IPhysicsEngine } from "../IPhysicsEngine";
+import type { IPhysicsEnginePluginV2 } from "./IPhysicsEnginePluginV2";
+
+import type { PhysicsRaycastResult } from "../physicsRaycastResult";
+import { _WarnImport } from "../../Misc/devTools";
+
+/**
+ * Class used to control physics engine
+ * @see https://doc.babylonjs.com/features/featuresDeepDive/physics/usingPhysicsEngine
+ */
+export class PhysicsEngineV2 implements IPhysicsEngine {
+    /**
+     * Global value used to control the smallest number supported by the simulation
+     */
+    public static Epsilon = 0.001;
+
+    //private _impostors: Array<PhysicsImpostor> = [];
+    //private _joints: Array<PhysicsImpostorJoint> = [];
+    private _subTimeStep: number = 0;
+    //private _uniqueIdCounter = 0;
+
+    /**
+     * Gets the gravity vector used by the simulation
+     */
+    public gravity: Vector3;
+
+    /**
+     * Factory used to create the default physics plugin.
+     * @returns The default physics plugin
+     */
+    public static DefaultPluginFactory(): IPhysicsEnginePluginV2 {
+        throw _WarnImport("CannonJSPlugin");

```

Comment:
No need mentioning any engine here (it implies cannon will get a new plugin, which is probably not going to happen)

Reply:
left empty for now